src/audit-logs/audit-logs.ts

```diff
@@ -0,0 +1,16 @@
+import { WorkOS } from '../workos';
+import { AuditLogEventOptions } from './interfaces';
+
+export class AuditLogs {
+  constructor(private readonly workos: WorkOS) {}
+
+  async createEvent(
+    organizationId: string,
+    event: AuditLogEventOptions,
+  ): Promise<void> {
+    await this.workos.post('/audit_logs/events', {
+      event,
```

Comment:
I think we'll need to stringify the Date in the event. How do we handle dates elsewhere? Do we just send the ISOString or a timestamp (unix time?) I prefer unix time, but we'll probably want to be consistent with other APIs.

Reply:
Axios does that actually.